### PR TITLE
Add new form link for domains and re-enable

### PR DESCRIPTION
**Pull request by rluodev**

### CODE REVIEW

**Review by devramsean0 — CHANGES_REQUESTED** [review]
You haven't re-enabled the perk card properly :)
Where you removed the `disabled` class. put this instead `#{"disabled" unless @perks_available}` This is important because the way you've got it currently will allow a user to apply for a domain even if their org isn't eligible. whereas by putting the "disabled" unless @perks_available it won't enable if the org isn't eligible

But otherwise LGTM

**Comment by hackliv** [comment]
Can we add another engineer to review this too!

**Comment by rluodev** [comment]
@cjdenio pls review thx!

**Comment by rluodev** [comment]
> You haven't re-enabled the perk card properly :) Where you removed the `disabled` class. put this instead `#{"disabled" unless @perks_available}` This is important because the way you've got it currently will allow a user to apply for a domain even if their org isn't eligible. whereas by putting the "disabled" unless @perks_available it won't enable if the org isn't eligible
> 
> But otherwise LGTM

fixed!

**Review by devramsean0 — APPROVED** [review]
Fixed! LGTM

File: app/views/events/promotions.html.erb [hunk 1]
```diff
@@ -171,8 +171,8 @@
           </div>
         <% else %>
           <%= link_to "Apply",
-            @perks_available ? airtable_form("shrKUmCPpbZ7Pdl0B", { "Project Name" => @event.name, "Bank URL" => event_url(@event), "Name" => current_user.name, "Login email" => current_user.email }) : root_path,
-            class: "btn bg-success mr1 disabled",
+            @perks_available ? fillout_form("pUUaNHEiT1us", { "hcbprojname" => @event.name, "hcburl" => event_url(@event), "name" => current_user.name, "email" => current_user.email }) : root_path,
```
Comment (cjdenio) [inline app/views/events/promotions.html.erb]:
```suggestion
            @perks_available ? fillout_form("pUUaNHEiT1us", { hcbprojname: @event.name, hcburl: event_url(@event), name: current_user.name, email: current_user.email }) : root_path,
```

**Review by cjdenio — APPROVED** [review]

**Pull request merged**